README.md

```diff
@@ -59,6 +59,28 @@ Note: Argument [quality] has a default value of 0 which corresponds to
 220-260 kbit/s VBR encoding.<br> 
 See [this article](https://wiki.hydrogenaudio.org/index.php?title=LAME#Recommended_encoder_settings) for more information.
 
+### flacer
+A script converting various audio files into a standard 16bit/44.1kHz flac format.<br>
+Usage:
+> flacer [filepath]
+
+Arguments:
+- filepath: path to the audio file to be converted
+
+### flactag
+Tool for metadata addition or editoin of flac filesm, automatically naming the file:
```

Comment:
typos: metadata edition*, files*;
Naming the file is not a part of the script's functionality - it has the form of a tagging wizard. Instructions need to reflect that somehow.

Reply:
It actually has, take a look at the line 63 in audiotoolbox.sh